examples/aws/terraform/ha-autoscale-cluster/ssm.tf

```
@@ -8,4 +8,5 @@ resource "aws_ssm_parameter" "license" {
   type      = "SecureString"
   value     = file(var.license_path)
   overwrite = true
+  tier      = "Advanced"
```

As per https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-advanced-parameters.html, I would use `Intelligent-Tiering` here so it can automatically pick the correct type. Changing this to `Advanced` will result in a resource recreation for folks who are currently using this code even though it might not be needed.

```suggestion
  tier      = "Intelligent-Tiering"
```